### PR TITLE
address review comments

### PR DESCRIPTION
**Pull request by fgimenez**
Followup from #8.

### CODE REVIEW

File: service/chartconfig/v1/appr/appr.go [hunk 1]
```diff
@@ -8,13 +8,15 @@ import (
...
+	Logger micrologger.Logger
+
 	Address string
 }
 
 // Client knows how to talk with a CNR server.
 type Client struct {
-	logger  micrologger.Logger
+	logger micrologger.Logger
+
```
Comment (xh3b4sd) [inline service/chartconfig/v1/appr/appr.go]: Nitpick, there is no block separation in line 34/35.

File: service/chartconfig/v1/resource/chart/resource.go [hunk 2]
```diff
@@ -16,16 +16,16 @@ const (
...
+	K8sClient  kubernetes.Interface
 	Logger     micrologger.Logger
 }
 
 // Resource implements the chart resource.
 type Resource struct {
 	// Dependencies.
-	k8sClient  kubernetes.Interface
 	apprClient *appr.Client
+	k8sClient  kubernetes.Interface
```
Comment (xh3b4sd) [inline service/chartconfig/v1/resource/chart/resource.go]: Nitpick, the sorting in the constructor is still "wrong".

File: service/chartconfig/v1/appr/appr.go (hunk 1 above)
Comment (fgimenez) [reply service/chartconfig/v1/appr/appr.go]: Ok done.

File: service/chartconfig/v1/resource/chart/resource.go (hunk 2 above)
Comment (fgimenez) [reply service/chartconfig/v1/resource/chart/resource.go]: Done.

**Pull request merged**